rts/Rendering/IconHandler.cpp

```diff
@@ -90,34 +96,49 @@ bool CIconHandler::AddIcon(
 	const std::string& texName,
 	float size,
 	float distance,
-	bool radAdj
+	bool radAdj,
+	float u0, float v0, float u1, float v1
 ) {
 	if (numIcons == iconData.size()) {
 		LOG_L(L_DEBUG, "[IconHandler::%s] too many icons added (maximum=%u)", __func__, numIcons);
 		return false;
 	}
 
 	unsigned int texID = 0;
-	unsigned int xsize = 0;
-	unsigned int ysize = 0;
 
 	bool ownTexture = true;
 
 	try {
-		CBitmap bitmap;
 
-		if ((ownTexture = !texName.empty() && bitmap.Load(texName))) {
-			texID = bitmap.CreateMipMapTexture();
+		ownTexture = false;
```

Comment:
is this var redundant now? always `false`

Reply:
Thanks, Ill fix that